src/pyshark/capture/pipe_capture.py

```diff
@@ -40,8 +40,10 @@ def get_parameters(self, packet_count=None):
         return params[:-1]
 
     def close(self):
-        # Close pipe
-        # self._pipe.close()  # Don't close the pipe. This should be the job of whatever is piping into it.
+        """
```

Comment:
You can actually just remove this entire method now, the docs in the classdoc should suffice